src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -341,16 +346,20 @@ public static void initProperties() {
 		}
 	}
 
-	public static String getProperty(String key) {
+	public String getProperty(String key) {
 		return getProperty(key, true);
 	}
 	
 	private static boolean missing(String s) {
 		return StringUtils.isEmpty(s) || "null".equals(s);
 	}
 
-	public static String getProperty(String key, boolean required) {
+	public String getProperty(String key, boolean required) {
 		initProperties();
```

Comment:
Is this loading the properties file on every `getProperty(..)` call?  do we need to init properties that many times?  I see about five calls.

Reply:
No, the properties file is not loaded on every `getProperty(...)` call.  Please see:
`	public void initProperties() {
		if (properties!=null) return; // <<< if the properties file is already loaded, don't do it again.
		properties = new Properties();
		...
}
`

Reply:
ahh ok. thanks.